src/includes/debug-symbols-apple/_default.mdx

```diff
@@ -167,7 +167,13 @@ You need to have an Auth Token for this to work. You can [create an Auth Token h
 1.  Download and install [sentry-cli](/product/cli/installation/)
 2.  You will need to copy the script below into a new _Run Script_ and set your _AUTH_TOKEN_, _ORG_SLUG_, and _PROJECT_SLUG_
 
-```bash
+<Alert level="" title="Warnings vs. errors">
+Depending on your needs, you may want to emit warnings to the Xcode build log to let the build pass, or emit errors to fail it, like in a CI deployment where debug symbols may be harder to recover, and you might not realize symbols woudn't be able to be symbolicated until after release.
```

Comment:
That sentence was too long, so I've broken it up

```suggestion
Depending on your needs, you may want to emit warnings to the Xcode build log to let the build pass, or emit errors to fail it. For example, in a CI deployment where debug symbols may be harder to recover, and you might not realize symbols couldn't be symbolicated until after release, it might be better to fail the build.
```